### ISSUE TITLE
SavedImageInfo: add ConsoleWriteFIlename()

### ISSUE THREAD

**Issue by swharden** [ScottPlot/ScottPlot#3943]
to allow something like this to print the full file path to the console

```cs
plot1.SavePng("test.png", 600, 400).ConsoleWriteFilename();
```

right now this works

```cs
var file = plot1.SavePng(saveAs, 600, 400);
Console.WriteLine(file.Path);
```

**Comment by swharden** [comment]
Also `CopyPathToClipboard()` and `CopyImageToClipboard()`

**Comment by swharden** [comment]
resolved in #3965
